azure-pipelines.yml

```diff
@@ -332,10 +332,6 @@ stages:
           - script: |
               pip install pytest pytest-cov pytest-azurepipelines
               pytest $(GE_pytest_opts) --napoleon-docstrings --junitxml=junit/test-results.xml --cov=. --cov-report=xml --cov-report=html --ignore=tests/cli --ignore=tests/integration/usage_statistics
-            env:
```

Comment:
Thanks for removing the `env` from here and the other places they are not needed!